vagrant/README.md

```diff
@@ -1,26 +1,34 @@
 # Prerequisites
 
 * [Download and install Vagrant.](https://www.vagrantup.com/) [More info about Vagrant here.](https://www.vagrantup.com/docs/)
-* [Virtualbox 5.0.X. (recommended)](https://www.virtualbox.org/wiki/Download_Old_Builds_5_0) (5.1 should work but if you're having issue, consider downgrading to 5.0)
+* Virtualbox ^5.2.X [Download here](https://www.virtualbox.org/wiki/Downloads)
+* `vagrant plugin install vagrant-docker-compose` (installs automatically)
 
 # Installation
 
-* `git clone git@github.com:eveseat/scripts.git`
+* Download eveseat/scripts `git clone git@github.com:eveseat/scripts.git`
```

Comment:
Mind updating this to the `https` link. I realise not everyone has an SSH key configured on Github.